### DISCUSSION TITLE
Is there a Roadmap for Restfox

### DISCUSSION THREAD

**Discussion by AnderssonPeter** [flawiddsouza/Restfox#75] [Q&A] [answered]
Hi, I'm trying to move away from Insomnia to some other Rest client.

But there are a few different things that I need to make my workflow work
* Have a Auth request that uses basic auth and generates a JWT Token, the result of this request needs to be usable in other requests.
* Input dialogs, this way I can input the ID in the request when I execute it instead of changing it on each request?

Is there a roadmap of planned features for Restfox?

**Comment by flawiddsouza** [comment]
Hi,

> Have a Auth request that uses basic auth and generates a JWT Token, the result of this request needs to be usable in other requests.

This one is already possible with plugins and environment variables. I'll provide an example on how to do it in a day or two when I have some time.

> Input dialogs, this way I can input the ID in the request when I execute it instead of changing it on each request?

Is there a client where this is implemented for reference? Definitely something I can pick up if you can give me some info on how this should work.

Currently there's no concrete roadmap as Restfox is feature complete as is. But I have some good things planned for the near future. Might list them out since this is the first time anyone has asked for that.

**Reply by AnderssonPeter** [reply]
Insomnia does this hope the images help.
![image](https://github.com/flawiddsouza/Restfox/assets/1308309/4e45fcc6-615f-43cb-8c04-499a01ea3bdb)
![image](https://github.com/flawiddsouza/Restfox/assets/1308309/1c1287b3-05dd-46bc-901b-bc9bb34e4713)

**Reply by flawiddsouza** [reply]
Thanks for the screenshots. This is helpful.

**Comment by flawiddsouza** [comment]
@AnderssonPeter Here's a video demo (https://www.youtube.com/watch?v=3cOQPm43Wus) to see to do this "Have a Auth request that uses basic auth and generates a JWT Token, the result of this request needs to be usable in other requests."

**Reply by AnderssonPeter** [reply]
Thanks looks like a good way to solve the issue :)

**Comment by gianpaj** [comment, marked as answer]
now there is https://github.com/flawiddsouza/Restfox/discussions/77
